RoaringBitmap/src/main/java/org/roaringbitmap/longlong/Roaring64Bitmap.java

```diff
@@ -348,15 +347,15 @@ public String toString() {
    * @return a custom iterator over set bits, the bits are traversed in ascending sorted order
    */
   @Override
-  public LongIterator getLongIterator() {
+  public PeekableLongIterator getLongIterator() {
     LeafNodeIterator leafNodeIterator = highLowContainer.highKeyLeafNodeIterator(false);
     return toIterator(leafNodeIterator, false);
   }
 
-  protected LongIterator toIterator(final LeafNodeIterator keyIte, boolean reverse) {
-    return new LongIterator() {
+  protected PeekableLongIterator toIterator(final LeafNodeIterator keyIte, boolean reverse) {
+    return new PeekableLongIterator() {
```

Comment:
This anonymous class is too large now , please rewrite it to an inner class.

Reply:
have split it into two separate classes one for each direction.